docs/source/settings.rst

```diff
@@ -82,7 +82,7 @@ You'll want to vary this a bit to find the best for your particular
 application's work load.
 
 By default, the value of the ``WEB_CONCURRENCY`` environment variable.
-If it is not defined, the default is 1.
+If it is not defined, the default is ``1``.
```

Comment:
This file is automatically generated. Please update `gunicorn/config.py` instead.

If you didn't touch this file and it was updated by `make html`, please revert all changes to it. I can do it in a separate commit.